python/kserve/kserve/protocol/rest/v1_endpoints.py

```diff
@@ -96,7 +96,7 @@ async def predict(self, model_name: str, request: Request) -> Union[Response, Di
             return Response(content=response, headers=response_headers)
         if isinstance(response, AsyncIterator):
             return StreamingResponse(content=response)
-        return JSONResponse(content=response, headers=response_headers)
+        return ORJSONResponse(content=response, headers=response_headers)
```

Comment:
ORJSONResponse is also much more performant than JSONResponse, is there any other place still using JSONResponse

Reply:
looks like these are the only places and by default it should be returning `ORJSONResponse`, see 
https://github.com/kserve/kserve/blob/e75491c986473ef86d66e5c0377a81de29e3462c/python/kserve/kserve/model_server.py#L188